pkg/reconciler/labeler/accessors.go

```diff
@@ -128,15 +129,19 @@ func updateRouteAnnotation(acc kmeta.Accessor, routeName string, diffAnn map[str
 			return
 		}
 		valSet.Delete(routeName)
-		diffAnn[serving.RoutesAnnotationKey] = strings.Join(valSet.UnsortedList(), ",")
+		sorted := valSet.UnsortedList()
```

Comment:
hm here it is not yet sorted, right? so maybe call that variable `routes` or something like that?

Reply:
yeah it is going to be sorted :) I can change it.